examples/atari_deepRL_saliency.ipynb

```diff
@@ -457,21 +457,6 @@
     "\n",
     "print_lock = threading.Lock() # Lock to control thread printing\n",
     "\n",
-    "\n",
-    "def postprocess_sal(sal, ff):\n",
-    "    \"\"\"\n",
-    "    Helper function for post-processing saliency maps, which includes resizing\n",
-    "    the saliency maps to (160,160) and multiplying them by a \"fudge factor\" ff\n",
-    "    as in the original implementation.\n",
-    "    \"\"\"\n",
-    "    pmax = sal.max()\n",
-    "    sal = cv2.resize(src=sal, dsize=(160,160)).astype(np.float32)\n",
-    "    sal -= sal.min()\n",
-    "    sal = ff * pmax * sal / sal.max()\n",
-    "    \n",
-    "    return sal\n",
```

Comment:
I thought we were trying to get rid of fudge-factors and the like with this revision? If you think it best to keep those in here for the sake of visualization, then OK, but we should be clear that this function is for that only. Ideally we would have a good explanation, or a link to one, for the effect of the fudge factor on the visualization, otherwise I don't know if it's just creating an illusions of success.

Reply:
I don't think it's possible to remove the fudge factor and still be faithful to the original implementation. However, I do believe that what Elim has now looks OK (without adding a fudge factor), so I'm happy to go with that if we don't think it's critical to match the original implementation exactly.